docs/environment.md

```diff
@@ -4,7 +4,7 @@ Log in to the LSF cluster, where all data processing must take place. Switch to
 
 Notes:
 * The first five variables are installation-specific and are blanked in this repository. You can get the values for the EVA use case from the [private repository](https://github.com/EBIvariation/configuration/blob/master/open-targets-configuration.md).
-* By modifying the `*REMOTE` and `*BRANCH` variables, you can run arbitrary versions of both the main and the VEP pipeline. This is highly useful for development and debugging. By default it fetches master branches of both repositories.
+* By modifying the `*REMOTE` and `*BRANCH` variables, you can run arbitrary versions of the pipeline. This is highly useful for development and debugging. By default it fetches master branches of both repositories.
```

Comment:
What's the current decision about the vep pipeline? integrated in this repo, right?

if so, there's only one repo and this sentence is outdated, 
> By default it fetches master branches of both repositories.

Reply:
Yes, the VEP pipeline is now integrated into this repository, and you're right, that was obsolete wording which I missed. Fixed in https://github.com/EBIvariation/eva-opentargets/pull/141/commits/9129d7b8f5f36b51a22069a2e278cef918ca1a29. Thanks!